lib/gooddata/lcm/actions/synchronize_user_filters.rb

```diff
@@ -121,6 +121,7 @@ def call(params)
             users_brick_input: params.users_brick_users
           }
           all_clients = domain.clients(:all, data_product).to_a
+          GoodData.gd_logger.info("Synchronizing in mode=#{mode}, number_of_clients=#{all_clients.size}, data_rows=#{user_filters.size}")
```

Comment:
Should we check user_filters.nil?

Reply:
No, We don't need check nil because user_filters already initial in load_data(...) method.